api/src/main/java/io/minio/MinioClient.java

```diff
@@ -967,6 +965,7 @@ private HttpResponse execute(Method method, String region, String bucketName, St
       } finally {
         response.body().close();
       }
+      scanner.close();
```

Comment:
Adding `scanner.close();` in `finally` is good enough than moving scanner out of `try` block.